extensions/open-remote-ssh/src/commands.ts

```diff
@@ -32,7 +32,7 @@ export async function promptOpenRemoteSSHWindow(reuseWindow: boolean) {
 	});
 
 	const quickPick = vscode.window.createQuickPick<HostQuickPickItem>();
-	quickPick.placeholder = vscode.l10n.t('Select a host, or type [user@]hostname[:port]');
+	quickPick.title = vscode.l10n.t('Select a host, or type [user@]hostname[:port]');
```

Comment:
We go from this

<img width="310" height="151" alt="Image" src="https://github.com/user-attachments/assets/030d7f34-eed6-4d5c-be92-d868bc6c632e" />

to this

<img width="483" height="176" alt="Image" src="https://github.com/user-attachments/assets/1e242ae4-de8b-4231-92b5-a67b5b9d3466" />

so that it's persistent.